bootstrap-conda

```diff
@@ -95,6 +95,7 @@ echo >&2 $0:$LINENO: generate conda environment files
         for pkg in $BOOTSTRAP_SYSTEM_PACKAGES; do
             echo "  - $pkg"
         done
+        echo "  - meson" 
```

Comment:
shouldn't this only go into src/environment-template?

Reply:
I guess you are right since in this mode we are still using make. 

Can we not delete the env files completely (along with the documentation)? Or at least move the conda env files that are currently in the root into the `build` dir, and then move the ones from `src` to the root?

Reply:
> Can we not delete the env files completely (along with the documentation)? 

We could, but it would not be an improvement.

> Or at least move the conda env files that are currently in the root into the `build` dir, and then move the ones from `src` to the root?

`src` is the root directory of the monolithic Sage library. There's a pyproject.toml there, and right next to it is the environment*.yml. That's exactly the right place for it.